Makefile

```diff
@@ -0,0 +1,48 @@
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+#
+# Copyright 2019 Joyent, Inc.
+#
+
+TOOLS =		deps/eng/tools
+NPM =		npm
+GIT =		git
+JSL =		deps/javascriptlint/build/install/jsl
+JSL_FLAGS =	--conf=$(TOOLS)/jsl.node.conf
+JSSTYLE =	deps/jsstyle/jsstyle
+JSSTYLE_FLAGS =	-f $(TOOLS)/jsstyle.conf -o indent=4
+
+JS_FILES := $(shell find lib test -name '*.js')
+
+# v8plus needs a little help
```

Comment:
> That comment is obsolete now?

I have two separate changes going:

- #1 includes TRITON-1941.  It modifies existing locker.js as little as possible (only a lint + copyright change) from what was found in sdc-cn-agent.  All of the tests, Makefile, etc. are new.
- #2 switches from node-lockfd to node-fs-ext for fcntl locking.

I broke it up this way because the initial implementation had no tests (or documentation) that I could find.  Before changing locking code that is scattered around Triton, I wanted to be real sure that before and after tests were in agreement as to how it worked.

The comment you asked about becomes obsolete with #2.